flocker/node/agents/ebs.py

```diff
@@ -60,10 +69,11 @@ def _blockdevicevolume_from_ebs_volume(ebs_volume):
 
     :return: Input volume in BlockDeviceVolume format.
     """
+    ebs_volume.update()
     return BlockDeviceVolume(
         blockdevice_id=unicode(ebs_volume.id),
         size=int(GB(ebs_volume.size).to_Byte().value),
-        attached_to=None,
+        attached_to=ebs_volume.attach_data.instance_id,
```

Comment:
I guess `attach_data` is set to something (not `None`) even when the volume isn't attached - or some tests would be failing when `create_volume` calls this function with an unattached volume.  Or maybe those tests would fail if I tried to run them on an EC2 instance.  Since we don't have buildbot set up to run these yet, I'll just ask - did you run the full ebs functional test suite and see it all pass?